src/classes/EP_EngagementPlans_UTIL.cls

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.